src/main/java/com/automatedtest/sample/pages/ImageTabOfSearchResultPage.java

```diff
@@ -0,0 +1,77 @@
+package com.automatedtest.sample.pages;
+
+import com.automatedtest.sample.utils.Constants;
+import org.junit.Assert;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.awt.*;
+import java.awt.event.InputEvent;
+import java.util.List;
+
+public class ImageTabOfSearchResultPage extends BasePage  {
+    @FindBy(xpath = "//input[@value='Search by image']")
+    WebElement buttonSearchByImage;
+
+    @FindBy(xpath = "//span[@aria-label='Search by image']")
+    WebElement spanSearchByImage;
+
+    @FindBy(xpath = "//img[starts-with(@src,'data:image') and @alt!='']/parent::div")
+    List<WebElement> listImageResult;
+
+    @FindBy(xpath = "//div[text()='Search by image']/parent::div/parent::div")
+    WebElement divToDropURL;
+
+    @FindBy(xpath = "//span[text()='Paste image URL']")
+    WebElement tabPasteURL;
+
+    public ImageTabOfSearchResultPage() {
+        PageFactory.initElements(this.driver, this);
+    }
+
+    public void dragNDropFirstImageToSearch() {
```

Comment:
if this is not working, either remove it or add a comment about how it doesn't work